src/_includes/page-footer.html

```diff
@@ -4,7 +4,7 @@
       <div class="col-sm-12 col-md-3">
         <div class="content">
           <div class="brand">
-            <img src="{% asset_path 'logo-white.svg' %}" alt="Dart logo" class="brand"/>
+            <img src="{% asset_path 'shared/dart/logo+text/horizontal/white.svg' %}" alt="Dart logo" class="brand"/>
```

Comment:
This logo seems less visible against the black footer than the old one, but I think we can fix this later (if at all).